src/uct/ib/rc/base/rc_ep.c

```diff
@@ -488,6 +493,84 @@ ucs_status_t uct_rc_ep_flush(uct_rc_ep_t *ep, int16_t max_available,
     return UCS_INPROGRESS;
 }
 
+static ucs_status_t uct_rc_ep_check_internal(uct_ep_h tl_ep)
+{
+    uct_rc_ep_t *ep         = ucs_derived_of(tl_ep, uct_rc_ep_t);
+    uct_rc_iface_t *iface   = ucs_derived_of(tl_ep->iface,
+                                            uct_rc_iface_t);
+    uct_rc_iface_ops_t *ops = ucs_derived_of(iface->super.ops, uct_rc_iface_ops_t);
+
+    /* in case if no TX resources are available then there is at least
+     * one signaled operation which provides actual peer status, in this case
+     * just return without any actions */
```

Comment:
this comment is not relevant for UCT_RC_CHECK_CQE_RET right? since it returns UCS_ERR_NO_RESOURCE

Reply:
added comment